### ISSUE TITLE
PHP Fatal error: Class 'LucaDegasperi\OAuth2Server\Storage\FluentStorageServiceProvider' not found

### ISSUE THREAD

**Issue by krues8dr** [lucadegasperi/oauth2-server-laravel#460]
Sorry, I know this is a dupe of #1, #303 and others, but I'm unsure of how to proceed.  I'm running Laravel 4.1, and the [Version Compatibility](https://github.com/lucadegasperi/oauth2-server-laravel#version-compability) table says that I'll need to use version 1 of the package to work, so I've got this in my `composer.json`:

```
"lucadegasperi/oauth2-server-laravel": "1.*",
```

But that results in the storage service being missing. Is there a more specific version or release I should be using?  Trying to install `3.*` fails due to missing `illuminate/http v4.2`.*.

**Comment by lucadegasperi** [comment]
The storage service in v1 is integrated in the normal service provider, you should be good to go with that.

**Comment by krues8dr** [comment]
Great, that worked, thanks!  Now I'm stuck at running the migrations, I'm getting 

```
  [InvalidArgumentException]
  There are no commands defined in the "oauth2-server" namespace.
```

I only see the `ExpiredTokensCommand` in the `Commands` directory of the package.

**Comment by krues8dr** [comment]
Ah, I just manually copied the migrations over and ran them and it seems to have worked.

**Comment by krues8dr** [comment]
So now I'm just missing the sample controller, I don't see one in the package.  I guess I should try one from the more recent packages.

**Comment by krues8dr** [comment]
I'm still struggling with this, after spending most of the day on it.  I've tried copying the 3.* Controller, but it needs the Authorizer helper which isn't present either, and that uses libraries that aren't included in the League oauth-server (v 2.1.3 ).  Is there a real, working example somewhere of using this on Laravel 4.1?  Maybe it'd be easier for me to just upgrade Laravel at this point.  :(

**Comment by krues8dr** [comment]
I've upgraded to 4.2 and things are working more properly.
